### PR TITLE
Add LinkedIn and Bluesky to the "Join" page

### PR DESCRIPTION
**Pull request by camlloyd**
This PR addresses https://github.com/nf-core/website/issues/2373, adding LinkedIn and Bluesky buttons to the "Join" page.
Bluesky is not available on FA yet but is due to be included in the next release. See: https://github.com/FortAwesome/Font-Awesome/issues/19810

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]
LGTM

**Review by christopher-hakkaart — APPROVED** [review]

**Review by maxulysse — CHANGES_REQUESTED** [review]
Super good, can you add some more details

File: src/pages/join.astro [hunk 1]
```diff
@@ -70,6 +81,20 @@ const headings = [
...
+            style="background-color: #0077B5;"
+        >
+            <i class="fab fa-linkedin"></i> LinkedIn
+        </a>
+        <a
+            href="https://bsky.app/profile/nf-co.re"
+            class="mb-2 me-3 btn btn-lg btn-dark"
+            style="background-color: #1185fe;"
+        >
+            Bluesky
```
Comment (ewels) [inline src/pages/join.astro:96]: Would be great to use the Bluesky logo, it's available on Iconify here: https://icon-sets.iconify.design/simple-icons/bluesky/

So should be able to use it with the `<Icon>` element, if added to `astro.config.mjs` as well.

File: package.json [hunk 2]
```diff
@@ -46,6 +46,7 @@
         "@nanostores/persistent": "^0.9.1",
         "astro": "^3.6.0",
         "astro-icon": "^1.0.2",
+        "astro-iconify": "^1.2.0",
```
Comment (mashehu) [inline package.json:49]: do we really need `astro-iconify` if we already have `astro-icon`

**Review by mashehu — CHANGES_REQUESTED** [review]

File: src/pages/join.astro [hunk 3]
```diff
@@ -41,6 +41,17 @@ const headings = [
...
+    {
+        text: 'LinkedIn',
+        slug: 'linkedin',
+        depth: 1,
+        fa_icon: 'fab fa-linkedin',
+    },
+    {
+        text: 'Bluesky',
+        slug: 'bluesky',
+        depth: 1,
```
Comment (mashehu) [inline src/pages/join.astro:53]: add icon here to show it in the toc

**Review by mashehu — APPROVED** [review]

**Comment by ewels** [comment]
I see some weird icons in the ToC on the deployment preview:

CleanShot 2024-03-25 at 09 33 39@2x

Otherwise looking good 👍🏻

**Comment by mashehu** [comment]
Fixed now

**Comment by mashehu** [comment]
@camlloyd hit merge, whenever you are happy 🙂

**Comment by ewels** [comment]
CleanShot 2024-03-25 at 11 09 06@2x

**Comment by camlloyd** [comment]
OK to squash?

**Pull request merged**